package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"scripts": {
+		"build": "tsc",
+		"start": "npm run build && node dist/cli.js",
+		"pretest": "npm run build",
+		"algo": "dist/cli.js validate -f test/specification.yml",
```

Comment:
```suggestion
		"algo": "npm run build && node dist/cli.js validate -f test/specification.yml",
```

Reply:
btw, I have to say I've never say approach with script called `algo`. You learn every single day 🤷🏼

Reply:
> <img alt="Screenshot 2021-04-16 at 13 55 38" width="645" src="https://user-images.githubusercontent.com/6995927/115020871-7e4d4c00-9ebb-11eb-999e-a05a5afd497d.png">
> 
> ![](https://camo.githubusercontent.com/e3fa59cedba4018b9735b3956465ede929a5304858ff29602a83df4b93cf5702/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f45326c4e7877586751774d3452364f6449752f67697068792e676966)
> 
> so what is missing, when do you feel comfortable to get some code review. I guess that just some cleanup is needed and we can also slowly start splitting the initial issue into separate issues for separate detaild discussion (if needed).
> 
> I want `asyncapi context -f <filepath>` so much that you can't imagine 😄

we can have a separate command namely context and have operations to add, remove or change the filename of the spec file. then if the context file is not set then we can throw an error or ask to set the context.

Reply:
Ouch! Just local test stuff that shouldn't be committed! 😅